app/models/job.ts

```diff
@@ -806,4 +812,17 @@ export class Job extends Record implements JobRecord {
     const links = this.links.filter((link) => link.rel === rel);
     return links.map(removeEmptyProperties) as JobLink[];
   }
+
+  /**
+   *  Computes and returns the date the data produced by the job will expire based on `createdAt`
```

Comment:
When I was testing this I immediately started thinking - should this instead be based on when the last file output file completed? I'm just thinking about future jobs that could take days or weeks to run.

I'm not sure what we would need to do to make sure the data doesn't get cleaned up before then - but I'm guessing we'll have to figure that out at some point to be able to extend expirations. Maybe revisit then?

Reply:
Yeah - I think it's important to have a single "guaranteed to last this long" date that should be good for the whole job, not just parts. Any data produced would last "at least" as long as this.